src/globals.css

```diff
@@ -272,10 +273,14 @@ body {
   border: none;
   width: 60%;
   display: block;
-  border-bottom: 0.3rem solid transparent;
+  /* border-bottom: 0.3rem solid transparent; */
```

Comment:
Remove this line instead of pushing the comment

Reply:
done